climt/__init__.py

```diff
@@ -12,7 +12,7 @@
     GridScaleCondensation, BergerSolarInsolation, SimplePhysics, RRTMGLongwave,
     RRTMGShortwave,
     EmanuelConvection, SlabSurface, GFSDynamicalCore,
-    DcmipInitialConditions, IceSheet, Instellation, DryConvectiveAdjustment)
+    DcmipInitialConditions, IceSheet, Instellation, DryConvectiveAdjustment, BucketSurface)
```

Comment:
Please rename the component to `BucketHydrology` instead of `BucketSurface`. This is a more commonly
used term in the field.